core/templates/dev/head/pages/learner_dashboard/learner_dashboard.html

```diff
@@ -440,6 +463,119 @@ <h1 class="stat-value"><[subscriptionsList.length]></h1>
+                  <span ng-if="thread.last_message_text"><[thread.last_message_text | truncate:50]></span>
+                  <span ng-if="!thread.last_message_text"><i>Suggestion</i></span>
+                <td class="col-lg-2 col-md-2 col-sm-2">
+                  <span><[getLocaleAbbreviatedDatetimeString(thread.last_updated)]></span>
+                </td>
+              </tr>
+            </table>
+
+            <table class="table" ng-if="feedbackThreadActive">
+              <tr>
+                <td>
+                  <button ng-click="backFromThread()"
+                          class="btn btn-default"
+                          style="padding: 0 3px;">
+                    <i class="material-icons oppia-vcenter" title="Return to list of messages"></i>
+                  </button>
+                  <span style="margin-left: 7px; font-size: 105%;"><[threadTitle]></span>
+                </td>
+              </tr>
+
+              <tr ng-repeat="message in messageSummaries" style="padding: 13px 8px 7px 8px;">
+                <td>
+                  <div class="row">
+                    <div class="col-sm-1 feedback-message-author-picture">
+                      <span>
+                        <img ng-src="<[message.author_picture_data_url]>"
+                             class="oppia-navbar-profile-picture img-circle"
+                             alt="User Avatar"
+                             style="margin-top: 4px;">
+                      </span>
+                    </div>
+                    <div class="col-sm-11">
+                      <div>
+                        <span><strong><[message.author_username]></strong></span>
+                      </div>
+
+                      <div>
+                        <span class="protractor-test-feedback-message" ng-if="message.text"><[message.text]></span>
+                          <button style="margin-top: 5px;" ng-if="message.current_content_html" class="btn btn-primary" ng-click="showSuggestionModal(message.suggestion_html, message.current_content_html, message.description)">
+                            View Suggestion
+                          </button>
+                      </div>
+                    </div>
+                  </div>
+                </td>
+              </tr>
+
+              <tr>
+                <td>
+                  <div class="row">
+                    <div class="col-sm-1 feedback-message-author-picture">
+                      <span>
+                        <img ng-src="<[profilePictureDataUrl]>"
+                             class="oppia-navbar-profile-picture img-circle"
+                             alt="User Avatar"
+                             style="margin-top: 4px;">
+                      </span>
+                    </div>
+
+                    <div class="col-sm-11">
+                      <div>
+                        <textarea class="form-control feedback-thread-reply-box" placeholder="Reply" ng-model="newMessage"></textarea>
```

Comment:
Should "reply" be i18n'd? Brings up an interesting question -- I guess we expect messages that people write to be in English? Does it depend on the language of the exploration?